Agents/ForecastingAgent/stack-manager-input-config/forecasting-agent.json

```diff
@@ -38,7 +38,7 @@
     },
     "endpoints": {
         "rest": {
-            "url": "http://localhost:5000/api/forecastingAgent/",
+            "url": "http://localhost:5005/api/forecastingAgent/",
```

Comment:
Please undo this change. It should stay at 5000. NB This is the internal number, not the external one that you changed in the docker compose files.